script.js

```diff
@@ -0,0 +1,63 @@
+//
+//
+// $("#box1").on('click' , function(){
+//   $("#box1").html("X");
+// });
+// $("#box2").on('click' , function(){
+//   $("#box2").html("O");
+// });
+// $("#box3").on('click' , function(){
+//   $("#box3").text("O");
+// });
+// $("#box4").on('click' , function(){
+//   $("#box4").text("O");
+// });
+// $("#box5").on('click' , function(){
+//   $("#box5").text("X");
+// });
+// $("#box6").on('click' , function(){
+//   $("#box6").text("X");
+// });
+// $("#box7").on('click' , function(){
+//   $("#box7").text("X");
+// });
+// $("#box8").on('click' , function(){
+//   $("#box8").text("O");
+// });
+// $("#box9").on('click' , function(){
+//   $("#box9").text("X");
+// });
+
+
+function reset() {
+
+  for (var i = 0; i < 9; i++) {
+    $(".box").eq(i).html("");
+  }
+
+}
+$('button').on("click", reset);
+
+
+//on click give values to box 
+
+var counter = 0;
+function sendX() {
+
+  for (var i = 0; i < 9; i++) {
+    $(".box").eq(i).on("click", function(){
+      // console.log("hello");
+      if (counter%2 === 0) {
+        $(this).html("X");
+        counter = counter +1;
+      }
+      else {
+        $(this).html("O");
+        counter = counter +1;
+      }
+
+    }
+  );}
+}
+sendX();
+// $(".box").on("click", sendX);
```

Comment:
I like the function, but what are lines 48 - 57 doing exactly? It seems like its defining some event listeners. That being said, is `sendX` a good name for this function?